rotkehlchen/db/upgrades/v40_v41.py

```diff
@@ -123,16 +125,63 @@ def _remove_covalent_api_key(write_cursor: 'DBCursor') -> None:
     log.debug('Exit _remove_covalent_api_key')
 
 
+def _move_labels_to_addressbook(write_cursor: 'DBCursor') -> None:
+    """Move all the `label` column values from `blockchain_accounts` table to the `name` column
+    of the 'address_book` table. If a `name` already exists in the `address_book` table, then
+    `address_name_priority` setting is used to determine which one to keep, defaults to
+    `DEFAULT_ADDRESS_NAME_PRIORITY`."""
+    log.debug('Enter _move_labels_to_addressbook')
+    address_name_priority = write_cursor.execute(  # get priority settings
+        'SELECT value FROM settings WHERE name = "address_name_priority"',
+    ).fetchone()
+    if address_name_priority is not None:
+        try:
+            address_name_priority = json.loads(address_name_priority[0])
+        except json.decoder.JSONDecodeError:
+            log.error(
+                'During v40->v41 DB upgrade a non-json address_name_priority setting was found: '
+                f'{address_name_priority[0]}. Reverting to default.',
+            )
+            address_name_priority = DEFAULT_ADDRESS_NAME_PRIORITY
+    else:
+        address_name_priority = DEFAULT_ADDRESS_NAME_PRIORITY
+
+    address_book_priority = address_name_priority.index('private_addressbook')
+    blockchain_account_priority = address_name_priority.index('blockchain_account')
+    labels_to_move = {  # get all the labels from `blockchain_accounts` table
+        (account, blockchain): label for account, blockchain, label in write_cursor.execute(
+            'SELECT account, blockchain, label FROM blockchain_accounts WHERE label IS NOT NULL',
+        )
+    }
+
+    if address_book_priority < blockchain_account_priority:
+        for address, blockchain in labels_to_move:
+            already_saved_name = write_cursor.execute(
+                'SELECT COUNT(*) FROM address_book WHERE address=? AND blockchain=?',
+                (address, blockchain),
+            ).fetchone()
+            if already_saved_name[0] > 0:  # replace with `address_book` names
+                labels_to_move[(address, blockchain)] = already_saved_name[0]
+    write_cursor.executemany(  # insert all the priotized labels into the `address_book` table
+        'INSERT OR IGNORE INTO address_book(address, blockchain, name) VALUES (?, ?, ?)',
```

Comment:
leave a space. Is hard to read if different things are one right after the other

```suggestion
                labels_to_move[(address, blockchain)] = already_saved_name[0]

    write_cursor.executemany(  # insert all the priotized labels into the `address_book` table
        'INSERT OR IGNORE INTO address_book(address, blockchain, name) VALUES (?, ?, ?)',
```